src/Workspaces/CoreTest/SyntaxReferenceTests.cs

```diff
@@ -69,7 +69,6 @@ public class C<>
             Assert.Equal(0, node.FullSpan.Length);
 
             var syntaxRef = tree.GetReference(node);
-            Assert.Equal("PathSyntaxReference", syntaxRef.GetType().Name);
```

Comment:
Why the heck did we have a test doing this...? :smile:

Reply:
that was probably our best way to go "yaeh... it's one of our special types that got created hre".  but yeah, ugly :)